zerver/views/user_groups.py

```diff
@@ -163,6 +166,9 @@ def edit_user_group(
     if description is not None and description != user_group.description:
         do_update_user_group_description(user_group, description, acting_user=user_profile)
 
+    if deactivated is not None and not deactivated and user_group.deactivated:
```

Comment:
Do we want to support deactivating groups by passing `deactivated` as `True` here along with the existing separate endpoint used for deactivation?

Reply:
I'm not sure; this is an API design discussion.